srcpkgs/ly/files/ly/run

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+TTY_NUM=2
```

Comment:
would it make sense to make that configurable via a `conf` file (see other run files for an example)

Reply:
It would make sense, yes. It unfortunately needs to be configured in ly's config files as well, so it will need to be manually synchronized.